src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1223,13 +990,19 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         // If the terminal translated the key, mark the event as handled.
         // This will prevent the system from trying to get the character out
         // of it and sending us a CharacterReceived event.
-        const auto handled = vkey ? _terminal->SendKeyEvent(vkey, scanCode, modifiers, keyDown) : true;
+        const auto handled = vkey ?

```

Comment:
gonna have to do a manual test pass to ensure that modifiers and OS-level keys do not jump the terminal down or dismiss selection.